nx/lib/nx/defn/grad.ex

```diff
@@ -498,6 +498,44 @@ defmodule Nx.Defn.Grad do
     [{input, dl}]
   end
 
+  defp grad(:qr, [{q, r}, input, _opts], ans, g) do
+    {q, r} = Nx.Defn.Expr.tuple({q, r}, ans)
```

Comment:
@josevalim this function doesn't exist anymore. What should I use here instead?

Reply:
@polvalente please make the function public again? The API is slightly different `.tuple(ans, [q, r])` but it should still be in Expr.